torch_xla/csrc/ir_util.cpp

```diff
@@ -85,16 +88,16 @@ std::vector<Value> Util::Clone(absl::Span<const Value> values,
 }
 
 std::vector<Value> Util::Clone(absl::Span<const Value> values) {
```

Comment:
Hmm, this can be done in a separate pr, can we verify if `Clone` is being used anywhere in our code base today? I think it is not. If that's the case we should consider removing it.

I think the motivation was we clone the IR graph and do some IR level optimization before lowering but that optimization was never done.

Reply:
Thanks! I'll keep this into a separate PR to make this PR concise as possible.